portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -58,4 +64,43 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     datastore.put(eventEntity);
     response.sendRedirect("/testForm.html");
   }
+  //TODO add doGet method
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    Query query = new Query("Event");
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    PreparedQuery results = datastore.prepare(query);
+        for (Entity entity : results.asIterable()){
+            String organizer = (String) entity.getProperty("organizer");
```

Comment:
looks like this whole section starting at the for loop is indented too far. the line with "for" should be indented to the same degree as the previous line